docs/deployment/README.md

```diff
@@ -260,6 +263,21 @@ dev:                            # deployment name
     elasticSearchMapping: 2         # Optional, triggers elasticSearch re-bootstrap.
                                     # Useful when e.g. mappings are updated.
 
+  # Optional, only necessary if EMS reports will be submitted to EMS
+  ems:
+    provider: CUMULUS
+    host: <ems-host>
+    path: <ems-report-path>
+    username: <ems-username>
+    dataSource: <report-datasource>
+    submitReport: true
+
+  # Optional, only necessary if Launchpad authentication is used for CMR and CUMULUS API
+  useLaunchpad: true
```

Comment:
Okay I also have something like this. Are we assuming that they'll switch to launchpad CMR and launchpad cumulus api auth at the same time? Should these be two different things? 
I think maybe different because for the Cumulus API they'll have to construct their own NAMS workflow and that will take time

Reply:
So you'll make it Clear this one is for the CMR tag

Reply:
Updated useLaunchpad to cmr.oauthProvider